### ISSUE TITLE
Issues in Activity Table detail

### ISSUE THREAD

**Issue by cfjedimaster** [ibm-functions/composer#8]
I ran fsh table and the initial display worked, but when I clicked into the detail, I got a blank screen (except for app UI). I thought maybe it was just slow wifi here (and if so, we need to add a loading indicator), but in DevTools I'm seeing errors:

```
Uncaught TypeError: bottomStripe.addModeButtons is not a function
    at C:\Users\ray\AppData\Roaming\npm\node_modules\@ibm-functions\shell\dist\IBM Cloud Functions Shell-win32-x64\resources\app.asar\content\js\bottom-stripe.js:97:39
    at capturedHeaders.forEach (C:\Users\ray\AppData\Roaming\npm\node_modules\@ibm-functions\shell\dist\IBM Cloud Functions Shell-win32-x64\resources\app.asar\content\js\picture-in-picture.js:59:13)
    at Array.forEach (native)
    at C:\Users\ray\AppData\Roaming\npm\node_modules\@ibm-functions\shell\dist\IBM Cloud Functions Shell-win32-x64\resources\app.asar\content\js\picture-in-picture.js:51:21
    at HTMLDivElement.backButton.onclick (C:\Users\ray\AppData\Roaming\npm\node_modules\@ibm-functions\shell\dist\IBM Cloud Functions Shell-win32-x64\resources\app.asar\content\js\picture-in-picture.js:106:13)
```

**Comment by cfjedimaster** [comment]
Ok - so one sec. The error above? I got this after clicking around a bit AFTER the initial black screen. I restarted, and the INITIAL error is a 404:

```
The requested resource could not be found. OpenWhiskError: GET https://openwhisk.ng.bluemix.net/api/v1/namespaces/_/actions/Space/serverless_superman/dotweet Returned HTTP 404 (Not Found) --> "The requested resource could not be found." Error
    at Client.handle_errors (C:\Users\ray\AppData\Roaming\npm\node_modules\@ibm-functions\shell\dist\IBM Cloud Functions Shell-win32-x64\resources\app.asar\plugins\node_modules\openwhisk\lib\client.js:137:11)
    at rp.catch.err (C:\Users\ray\AppData\Roaming\npm\node_modules\@ibm-functions\shell\dist\IBM Cloud Functions Shell-win32-x64\resources\app.asar\plugins\node_modules\openwhisk\lib\client.js:96:42)
    at tryCatcher (C:\Users\ray\AppData\Roaming\npm\node_modules\@ibm-functions\shell\dist\IBM Cloud Functions Shell-win32-x64\resources\app.asar\plugins\node_modules\bluebird\js\main\util.js:26:23)
    at Promise._settlePromiseFromHandler (C:\Users\ray\AppData\Roaming\npm\node_modules\@ibm-functions\shell\dist\IBM Cloud Functions Shell-win32-x64\resources\app.asar\plugins\node_modules\bluebird\js\main\promise.js:510:31)
    at Promise._settlePromiseAt (C:\Users\ray\AppData\Roaming\npm\node_modules\@ibm-functions\shell\dist\IBM Cloud Functions Shell-win32-x64\resources\app.asar\plugins\node_modules\bluebird\js\main\promise.js:584:18)
    at Promise._settlePromises (C:\Users\ray\AppData\Roaming\npm\node_modules\@ibm-functions\shell\dist\IBM Cloud Functions Shell-win32-x64\resources\app.asar\plugins\node_modules\bluebird\js\main\promise.js:700:14)
    at Async._drainQueue (C:\Users\ray\AppData\Roaming\npm\node_modules\@ibm-functions\shell\dist\IBM Cloud Functions Shell-win32-x64\resources\app.asar\plugins\node_modules\bluebird\js\main\async.js:123:16)
    at Async._drainQueues (C:\Users\ray\AppData\Roaming\npm\node_modules\@ibm-functions\shell\dist\IBM Cloud Functions Shell-win32-x64\resources\app.asar\plugins\node_modules\bluebird\js\main\async.js:133:10)
    at MutationObserver.Async.drainQueues (C:\Users\ray\AppData\Roaming\npm\node_modules\@ibm-functions\shell\dist\IBM Cloud Functions Shell-win32-x64\resources\app.asar\plugins\node_modules\bluebird\js\main\async.js:15:14)
    at process._tickCallback (internal/process/next_tick.js:103:7) OpenWhiskError {name: "OpenWhiskError", message: "GET https://openwhisk.ng.bluemix.net/api/v1/namesp… --> "The requested resource could not be found."", error: Object, statusCode: 404, stack: "Error↵    at Client.handle_errors (C:\Users\ray\Ap…ickCallback (internal/process/next_tick.js:103:7)"}
err @ ui.js:412
cmdPromise.then.then.catch.err @ C:\Users\ray\AppData\Roaming\npm\node_modules\@ibm-functions\shell\dist\IBM Cloud Functions Shell-w…:636
_tickCallback @ internal/process/next_tick.js:103
```

**Comment by cfjedimaster** [comment]
Ahah! It's the space in my space? See in the URL how it is:

"Space"

My actual space is "My Space". I bet that's it.

**Comment by cfjedimaster** [comment]
Or maybe it's something else. In DevTools, I copied the 404 URL, added "My%20" in front of space, logged in with the right values, and it was still bad. Removing /Space though fixed it.

**Comment by starpit** [comment]
this should be fixed in 1.3.272

**Comment by cfjedimaster** [comment]
confirmed (don't want to close as it's not my project :)
